sdk/qumulo/Azure.ResourceManager.Qumulo/api/Azure.ResourceManager.Qumulo.net8.0.cs

```diff
@@ -59,18 +59,9 @@ protected QumuloFileSystemResourceCollection() { }
     }
     public partial class QumuloFileSystemResourceData : Azure.ResourceManager.Models.TrackedResourceData, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Qumulo.QumuloFileSystemResourceData>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Qumulo.QumuloFileSystemResourceData>
     {
-        public QumuloFileSystemResourceData(Azure.Core.AzureLocation location, Azure.ResourceManager.Qumulo.Models.MarketplaceDetails marketplaceDetails, Azure.ResourceManager.Qumulo.Models.StorageSku storageSku, Azure.ResourceManager.Qumulo.Models.QumuloUserDetails userDetails, string delegatedSubnetId, string adminPassword, int initialCapacity) { }
-        public string AdminPassword { get { throw null; } set { } }
-        public string AvailabilityZone { get { throw null; } set { } }
-        public System.Uri ClusterLoginUri { get { throw null; } set { } }
-        public string DelegatedSubnetId { get { throw null; } set { } }
+        public QumuloFileSystemResourceData(Azure.Core.AzureLocation location) { }
         public Azure.ResourceManager.Models.ManagedServiceIdentity Identity { get { throw null; } set { } }
-        public int InitialCapacity { get { throw null; } set { } }
-        public Azure.ResourceManager.Qumulo.Models.MarketplaceDetails MarketplaceDetails { get { throw null; } set { } }
-        public System.Collections.Generic.IList<System.Net.IPAddress> PrivateIPs { get { throw null; } }
-        public Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState? ProvisioningState { get { throw null; } }
-        public Azure.ResourceManager.Qumulo.Models.StorageSku StorageSku { get { throw null; } set { } }
-        public string UserDetailsEmail { get { throw null; } set { } }
+        public Azure.ResourceManager.Qumulo.Models.QumuloFileSystemResourceProperties Properties { get { throw null; } set { } }
```

Comment:
so this is causing a breaking change - since this package has been released a stable version, we need to change this back.

We need to add a `flattenProperty` decorator to its corresponding `properties` property, try to add something like this in your `client.tsp`:
```
@@flattenProperty(FileSystemResource.properties, "csharp");
```
Please update the model name if this is incorrect because I do not know its exact name in your spec.
A hint that you could search this first to see if someone has added one statement about this, if so, we just need to add `csharp` to the scope.